src/r-bridge/shell.ts

```diff
@@ -53,7 +57,8 @@ export const DEFAULT_OUTPUT_COLLECTOR_CONFIGURATION: OutputCollectorConfiguratio
 	postamble: `🐧${'-'.repeat(5)}🐧`,
 	timeout:   {
 		ms:             750_000,
-		resetOnNewData: true
+		resetOnNewData: true,
+		reject:         true
```

Comment:
i am not such a big fan of a `reject` flag. how about a more general purpose `onError` handler?

Reply:
fiiine